### ISSUE TITLE
Self-Play and Self-Describe: Policy Adaptation with Vision-Language Foundation Models. (arXiv:2212.07398v1 [cs.LG])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#23996]
Self-Play and Self-Describe: Policy Adaptation with Vision-Language Foundation Models. (arXiv:2212.07398v1 [cs.LG])<br>
https://ift.tt/k8qguEQ<br>
<p>Recent progress on vision-language foundation models have brought significant advancement to building general-purpose robots. By using the pre-trained models to encode the scene and instructions as inputs for decision making, the instruction-conditioned policy can generalize across different objects and tasks. While this is encouraging, the policy still fails in most cases given an unseen task or environment. To adapt the policy to unseen tasks and environments, we explore a new paradigm on leveraging the pre-trained foundation models with Self-PLAY and Self-Describe (SPLAYD). When deploying the trained policy to a new task or a new environment, we first let the policy self-play with randomly generated instructions to record the demonstrations. While the execution could be wrong, we can use the pre-trained foundation models to accurately self-describe (i.e., re-label or classify) the demonstrations. This automatically provides new pairs of demonstration-instruction data for policy fine-tuning. We evaluate our method on a broad range of experiments with the focus on generalization on unseen objects, unseen tasks, unseen environments, and sim-to-real transfer. We show SPLAYD improves baselines by a large margin in all cases. Our project page is available at https://ift.tt/6iB3IWX</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
